README.md

```diff
@@ -957,6 +957,7 @@ Simplified file servers for sharing of one-time/short-lived/temporary files, pro
 - [Snapdrop](https://snapdrop.net/) - Local file sharing in your browser. Inspired by Apple's Airdrop. ([Source Code](https://github.com/RobinLinus/snapdrop)) `GPL-3.0-only` `Docker`
 - [transfer.sh](https://transfer.sh) - Easy file sharing from the command line. ([Source Code](https://github.com/dutchcoders/transfer.sh)) `MIT` `Go`
 - [Uguu](https://uguu.se/) - Stores files and deletes after X amount of time. ([Source Code](https://github.com/nokonoko/uguu)) `MIT` `PHP`
+- [Uploady](https://github.com/farisc0de/Uploady) - Uploady is a Simple File Uploader Script with Multi File Upload Support. `MIT` `PHP`
```

Comment:
one quick chanage to be made!

please change the description to sentence case

Reply:
Sorry, what a careless mistake!